db/seeds.rb

```diff
@@ -7,3 +7,56 @@
 #   ["Action", "Comedy", "Drama", "Horror"].each do |genre_name|
 #     MovieGenre.find_or_create_by!(name: genre_name)
 #   end
+30.times do
+  Trip.create!({
+    name: Faker::Name.name,
+    location: Faker::Address.country,
+    start_date: Faker::Time.between(from: DateTime.now, to: DateTime.now + 50),
+    end_date: Faker::Time.between(from: DateTime.now + 51, to: DateTime.now + 75),
+    status: Faker::Number.between(from: 0, to: 1),
+    total_budget: Faker::Number.between(from: 500, to: 10000),
+    user_id: Faker::Number.between(from: 1000, to: 1005)
+  })
```

Comment:
I set the user_id to a high number so it doesn't conflict with new users made on the FE and apply data that is incorrect